documentation/source/code-snippets/fixed-term-fixed-rate-repo-product.json

```diff
@@ -0,0 +1,202 @@
+{
+	"product": {
+		"contractualProduct": {
+			"productTaxonomy": [
+				{
+					"source": "CFI",
+					"value": {
+						"name": {
+							"value": "LRSTXD"
+						}
+					}
+				},
+				{
+					"productQualifier": "Repurchase Agreement",
+					"taxonomySource": "CFI"
+				}
+			],
+			"economicTerms": {
+				"effectiveDate": {
+					"adjustableDate": {
+						"meta": {
+							"globalKey": "3f28d3"
```

Comment:
(Suggestion only): remove all the "meta" / "globalKey" attributes from the snippet, as it makes it unnecessarily verbose - unless its a necessary cross-reference that could destroy the object's integrity. I think only "RepurchaseDate" falls in that bucket.

Reply:
@tomhealey-icma I have updated this